test/functional/viml/lang_spec.lua

```diff
@@ -0,0 +1,22 @@
+local helpers = require('test.functional.helpers')(after_each)
+local clear, eval, eq = helpers.clear, helpers.eval, helpers.eq
+local execute, source = helpers.execute, helpers.source
+
+describe('turkish', function()
+  before_each(clear)
+
+  it('applies locale to \'i\' in `<SID>` comparison', function()
+    execute('lang ctype tr_TR.UTF-8')
+    if string.find(eval('v:errmsg'), '^E197: ') then
+      pending("Locale tr_TR.UTF-8 not supported")
```

Comment:
Should have a `return` after the pending line so we don't execute the rest of the test.